burnman/output_seismo.py

```diff
@@ -0,0 +1,280 @@
+# This file is part of BurnMan - a thermoelastic and thermodynamic toolkit for the Earth and Planetary Sciences
+# Copyright (C) 2012 - 2015 by the BurnMan team, released under the GNU
+# GPL v2 or later.
+
+
+from __future__ import absolute_import
+
+import numpy as np
+import warnings
+import scipy.integrate
+import matplotlib.pyplot as plt
+import pkgutil
+
+from . import tools
+from . import constants
+from . import seismic
+from . import geotherm
+
+def write_axisem_input(rock, min_depth=670.e3, max_depth=2890.e3, T0= 1900, filename='axisem_burnmantestrock.txt',
+                       axisem_ref='axisem_prem_ani_noocean.txt', plotting=False):
+    """
+    Writing velocities and densities to AXISEM (www.axisem.info) input file
+    Default is set to replacing the lower mantle with the BurnMan rock
+    Note:
+        - This implementation uses PREM to convert from depths to pressures to compute at
+        - This implementation assumes an adiabatic temperature profile, only T0 at min_depth can be set
+        - Currently, it only honors the discontinuities already in the synthetic input file, so it is best 
+            to only replace certain layers with burnman values (this should be improved in the future).
+            
+            
+    Parameters
+    ----------
+    rock  :  burnman.Composite()
+        Composition to implement in the model
+    min_depth : float
+        minimum depth to replace model (m) (default = 670 km)
+    max_depth : float 
+        minimum depth to replace model (m) (default = 2890 km)
+    T0 : float
+        Anchor temperature at min_depth for adiabatic profile (K) (default=1900)
+    filename: string
+        Output filename (default ='axisem_burnmantestrock.txt')
+    axisem_ref: string
+        Input filename (in burnman/data/input_seismic/) (default = 'axisem_prem_ani_noocean.txt')
+    plotting: Boolean
+        True means plot of the old model and replaced model will be shown (default = False)
+
+    """
+
+
+    
+    # Load reference input
+    datastream = pkgutil.get_data('burnman', 'data/input_seismic/' + axisem_ref)
+    lines = [line.strip()
+                 for line in datastream.decode('ascii').split('\n') if line.strip()]
+    table = []
+    for line in lines[18:]:
```

Comment:
I don't particularly like skipping exactly 17 lines, because it is not very robust. You could for example skip every line until you encounter one that starts with ``COLUMNS``.